.gitignore

```diff
@@ -24,3 +24,9 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+/config/credentials/development.key
+
+/config/credentials/test.key
+
+/config/credentials/environment.key
```

Comment:
shouldn't this be `production.key`? 🤔

Reply:
That file was created by mistake and it has been removed !! Thanks 👍